respringless_tweaks.md

```diff
@@ -0,0 +1,152 @@
+# How do you create a respringless tweak?
+
+* It's actually really easy if you understand the basics of it. What I'll make is a simple tweak that adds a blur to the LockScreen, with the option to change the its alpha/intensity on the fly. Think about it, how could that happen without a respring? Simple, Preferences needs to communicate with SpringBoard to notify that the blur value has been changed, so that it can refresh the UI. The `NSNotificationCenter` Foundation API makes that really straightforward.
```

Comment:
```suggestion
*What we'll make is a simple tweak that adds a blur to the lock screen, with the option to change the its alpha/intensity on the fly. How can this happen without a respring? The Settings app will need to communicate with SpringBoard to notify that the blur value has been changed, so that it can refresh the UI. The `NSNotificationCenter` Foundation API makes that really straightforward.
```